common/primitives/src/schema.rs

```diff
@@ -51,8 +51,8 @@ pub struct SchemaResponse {
 	/// The unique identifier for this Schema
 	pub schema_id: SchemaId,
 	/// The data that represents how this schema is structured
-	#[cfg_attr(feature = "std", serde(with = "as_string"))]
-	pub model: Vec<u8>,
+	#[cfg_attr(feature = "std", serde(with = "as_model"))]
+	pub model: ParquetModel,
```

Comment:
what if it's not parquet?

Reply:
Good point, this part isn't ready yet. Ideally we'd use a trait here, but bc of the dynamic memory allocation associated with traits, we'd have to either define an `impl` for a dynamic trait or use an enum. I'm investigating both approaches.